common/src/main/scala/common/util/VersionUtil.scala

```diff
@@ -52,7 +52,7 @@ object VersionUtil {
   /**
     * A regex compatible with the dependency constants in project/Dependencies.scala.
     */
-  private val SbtDependencyVersionRegex = """\s+private val (\w+)V = "([^"]+)"""".r
+  private val SbtDependencyVersionRegex = """\s+private val (\w+)V = "([^"]+)".*""".r
```

Comment:
Stowaway change?

Reply:
Yeah. If one runs `sbt clean` the version confs get wiped out. IntelliJ compilation doesn't do resource generation. So if one is running locally they won't have the conf files used to tell the code what version of the swagger-ui is being used. Without that version information, the code doesn't know where to load the embedded index.html files from inside the swagger-ui JAR file.

This bit of code used to FTFY IntellIJ by ~hacking~ looking up the swagger-ui version from Dependencies.scala.

Except recently, instead of:

```scala
private val swaggerUiV = "3.23.11"
```

For some _unknown reason_ dependencies have started to look like:

```scala
private val swaggerUiV = "3.23.11" // scala-steward:off
```

So this regex ~hack~ lookup for IntelliJ had to be updated to account for extra-stuffs after the version.